frontend/src/utils/utils.tsx

```diff
@@ -0,0 +1,72 @@
+import { ILookupCode } from 'actions/lookupActions';
```

Comment:
Not a biggie but ideally `*.tsx` should be reserved to components that output JSX markup. This file doesn't produce any JSX (yet) so I renamed to `*.ts` in a previous PR. I'm not against of everything being *.tsx - just want to ask the team what they prefer.

Reply:
I have no strong preferences, only did this to resolve some warnings I was seeing.